programs/amm/src/libraries/liquidity_math.rs

```
@@ -220,16 +236,21 @@ pub fn get_delta_amount_1_unsigned(
             U256::from(fixed_point_64::Q64),
         )
     }
-    .unwrap()
-    .as_u64()
+    .unwrap();
```

Are we ok to always unwrap here? Alternatively we can also throw an error.